kvm_gen/src/x86/bindings.rs

```diff
@@ -267,9 +267,9 @@ pub const KVM_S390_RESET_CLEAR: ::std::os::raw::c_uint = 2;
 pub const KVM_S390_RESET_SUBSYSTEM: ::std::os::raw::c_uint = 4;
 pub const KVM_S390_RESET_CPU_INIT: ::std::os::raw::c_uint = 8;
 pub const KVM_S390_RESET_IPL: ::std::os::raw::c_uint = 16;
-pub const KVM_SYSTEM_EVENT_SHUTDOWN: ::std::os::raw::c_uint = 1;
-pub const KVM_SYSTEM_EVENT_RESET: ::std::os::raw::c_uint = 2;
-pub const KVM_SYSTEM_EVENT_CRASH: ::std::os::raw::c_uint = 3;
+pub const kvm_genTEM_EVENT_SHUTDOWN: ::std::os::raw::c_uint = 1;
```

Comment:
It looks like these lines were affected by some sort of find/replace collateral damage.

Reply:
@alexandruag Fixed.